src/apps/openfluid-devstudio/MainWindow.hpp

```diff
@@ -43,10 +43,14 @@
 
 #include <QMap>
 
+#include <apps/openfluid-devstudio/WorkspaceToolbar.hpp>
```

Comment:
<apps/openfluid-devstudio/WorkspaceToolbar.hpp> is not in the OpenFLUID framework, use "WorkspaceToolbar.hpp" instead for inclusion, then put it after inclusions ofstd/frameworks headers
see https://community.openfluid-project.org/scidev/codestyle/#included-header-files